liqd_product/assets/scss/_form.scss

```diff
@@ -99,8 +99,6 @@ div.cke_focus {
 }
 
 .form-check__label {
-    padding-left: 1.25em;
-
```

Comment:
@fuzzylogic2000 I know there was a reason we don't do this but I clicked through and couldn't see anywhere it went wrong, might check again in morning

Reply:
found it, yeah can't do this

Reply:
Did it change how the stuff is displayed, when a chcekbox is hidden by a button? like here in the poll-dashboard:
![Screenshot from 2019-10-23 14-52-11](https://user-images.githubusercontent.com/8178179/67394636-d1835e80-f5a4-11e9-8d78-88617bcd2316.png)

Reply:
@fuzzylogic2000 I'm not sure but it messed up the label choice for the user so I reverted the change, I'll add in the issue that its a longer fix and has always been like that, if its a priority then I can add additional class names and do some nested styles but as currently check boxes are the same everywhere I am not overly keen and only changing them in 1 place.

Reply:
No, that wasn't it, that's just currently broken...

Reply:
That's a problem from the old grid system, i'll fix with this pull request